content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -9,176 +9,268 @@ With a down payment, the buyer pays a portion of the total amount owed while agr
-and add a tax amount, if necessary, in the :guilabel:`Customer Taxes` field. These settings will be
-reused for future down payments.
+On the :guilabel:`Create invoices` pop-up form, the down payment options are:
 
-.. image:: down_payment/create-invoices-popup-form.png
+- :guilabel:`Down payment (percentage)`
+- :guilabel:`Down payment (fixed amount)`
+
+Once the desired down payment option is selected in the :guilabel:`Create Invoice` field on the
+pop-up form, designate the desired amount (either as a percentage or fixed amount) in the
+:guilabel:`Down Payment Amount` field.
+
+Then, select the appropriate income account for the invoice in the :guilabel:`Income Account` field.
+Then, select a tax amount, if necessary, in the :guilabel:`Customer Taxes` drop-down field.
+
+.. image:: down_payment/create-invoices-popup-form-filled-out.png
    :align: center
-   :alt: How to configure a down payment on Odoo Sales.
+   :alt: A create invoices pop-up form with down payment fields filled in with information.
 
-.. important::
-   When a down payment is first requested, a new product called :guilabel:`Down payment` is created.
-   This product is registered as a :guilabel:`Service` product, with an invoicing policy of
-   :guilabel:`Ordered quantities`. This product can be edited and modified at any time.
+When all fields are filled in with the desired information, click the :guilabel:`Create Draft
+Invoice` button. Upon clicking :guilabel:`Create Draft Invoice` on the :guilabel:`Create invoices`
+pop-up form, Odoo reveals the :guilabel:`Customer Invoice Draft`.
+
+In the :guilabel:`Invoice Lines` tab of the :guilabel:`Customer Invoice Draft` the down payment that
+was just configured in the :guilabel:`Create invoices` pop-up form appears as a :guilabel:`Product`.
+
+.. image:: down_payment/down-payment-product-invoice-draft.png
+   :align: center
+   :alt: Down payment as a product in the invoice lines tab of a customer invoice draft in Odoo.
+
+.. note::
+   When the :guilabel:`Down payment` product in the :guilabel:`Invoice Lines` tab is clicked, Odoo
+   reveals the product form for the down payment.
+
+   By default, the :guilabel:`Product Type` of down payment products generated for invoices are set
+   as :guilabel:`Service`, with the :guilabel:`Invoicing Policy` set to :guilabel:`Prepaid/Fixed
+   Price`.
+
+   .. image:: down_payment/down-payment-product-form.png
+      :align: center
+      :alt: Down payment product form with service product type and invoicing policy field.
+
+   This product can be edited/modified at any time.
 
 .. warning::
-   If :guilabel:`Delivered quantities` is chosen as the invoicing policy, an invoice will **not** be
-   able to be created.
+   If :guilabel:`Based on Delivered Quantity (Manual)` is chosen as the :guilabel:`Invoicing
+   Policy`, an invoice will **not** be able to be created.
 
-Request a 50% down payment
-==========================
+Request 50% down payment (example)
```

Comment:
```suggestion
Example: request 50% down payment
```
Just a suggestion to bring the word "example" to the front. Feel free to ignore if you like it as is